SmartDeviceLink/private/SDLVoiceCommandUpdateOperation.m

```diff
@@ -155,6 +157,17 @@ - (void)sdl_sendCurrentVoiceCommands:(void(^)(void))completionHandler {
     return [mutableDeletes copy];
 }
 
+/// Remove a voice command from the array of voice commands if it has no strings
+- (NSArray<SDLVoiceCommand *> *)sdl_removeEmptyVoiceCommands {
+    NSMutableArray<SDLVoiceCommand *> *pendingVoiceCommands = [[NSMutableArray alloc] init];
+    for (SDLVoiceCommand *voiceCommand in self.pendingVoiceCommands) {
+        if (voiceCommand.voiceCommands.count > 0) {
```

Comment:
We should also add a check if any if the `voiceCommands` are empty strings (i.e. `@""`) since the module will reject empty strings for `voiceCommands` (https://github.com/smartdevicelink/rpc_spec/blob/8b8c996c738711c4235c8574c179354cbf8982e2/MOBILE_API.xml#L1196)